src/stan/analyze/mcmc/compute_effective_sample_size.hpp

```diff
@@ -35,6 +37,30 @@ namespace analyze {
       num_draws = std::min(num_draws, sizes[chain]);
     }
 
+    if ( num_draws < 3 ) {
```

Comment:
This edge case should be noted in the function documentation.  Perhaps also a note here about why less than three (e.g. variance cannot be estimated for less than three draws).

Reply:
Done